File: otto-mobile/otto-mobile/PhotoUploadLimits.swift
import Foundation

enum PhotoUploadLimits {
    /// Matches server multer cap across avatar, garage, squad, event banner, and chat photos.
    static let serverMaxBytes = 24 * 1024 * 1024
    /// Keep below the server cap, leaving room for multipart overhead.
    static let clientTargetMaxBytes = serverMaxBytes - 500_000
}

File: otto-mobile/otto-android-LEGACY-DO-NOT-OPEN/app/src/main/java/to/ottomot/driftd/core/data/PhotoUploadLimits.kt
package to.ottomot.driftd.core.data

/** Matches server multer cap across avatar, garage, squad, event banner, and chat photos. */
internal const val PhotoUploadMaxBytes: Int = 24 * 1024 * 1024

/** Slightly below [PhotoUploadMaxBytes] for multipart overhead (parity with iOS). */
internal const val PhotoUploadClientTargetMaxBytes: Int = PhotoUploadMaxBytes - 500_000
